### DISCUSSION TITLE
impl headers

### DISCUSSION THREAD

**Discussion by goiw111** [ntex-rs/ntex#121] [Ideas]
I worked on some headers, can you check the commit, please
https://github.com/goiw111/ntex/commit/935e63cf59fc040d404643f06e7f82e57e499a2d

**Comment by fafhrd91** [comment]
looks good. i am not sure if it us belongs to ntex though

**Reply by goiw111** [reply]
I thought it will be useful in ntex::web. anyway, I like all the work you made in that repo or actix-web and I want to be a part of that community.

**Reply by fafhrd91** [reply]
ok, sounds good. create pr then.

I don't think there is any community around ntex, so do not expect much attention.
